### DISCUSSION TITLE
Heyu on Raspberry Pi with CM11A

### DISCUSSION THREAD

**Discussion by ghutib** [HeyuX10Automation/heyu#45] [Q&A] [answered]
Folks,

After many years of running Heyu with a CM11A successfully on a CentOS 7 platform, I need to migrate Heyu to a Raspberry Pi (a 3B, if that matters).

So I'll need to acquire a USB to RS232 adapter. I've read that some folks have had problems with some USB to RS232 adapters with Heyu/Raspberry Pi and CM11A.

Is there a specific USB/serial adapter that is known to work with Heyu and CM11A on a Raspberry Pi?

And/or are there specific USB/serial adapter chipsets that are known to work with Heyu and CM11A on a Raspberry Pi?

Thanks.

Regards,
ghutib

**Comment by davepet** [comment, marked as answer]
I have great results using this adapter with Rpi3b and CM11A or CM17.

Plugable USB to Serial Adapter Compatible with Windows, Mac, Linux (RS-232DB9 Female Connector, 
Prolific PL2303HX Rev. D Chipset)
